src/main/java/ProductPage.java

```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ProductPage {
+
+    WebDriver driver;
+    public static final String idTextDispayed = "productTitle";
+
+    public ProductPage(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public boolean isNintendoSwitchContained(String expectedText){
```

Comment:
can you modify this function to return just the text, so that we do the contains in the test?